src/temp/ProjectInfo/InfoContainer.tsx

```diff
@@ -0,0 +1,143 @@
+    }
+    return newDateArr.join('-');
+  };
+
+  const seasons = [
+    { id: 0, title: ready ? t('common:january') : '' },
+    { id: 1, title: ready ? t('common:february') : '' },
+    { id: 2, title: ready ? t('common:march') : '' },
+    { id: 3, title: ready ? t('common:april') : '' },
+    { id: 4, title: ready ? t('common:may') : '' },
+    { id: 5, title: ready ? t('common:june') : '' },
+    { id: 6, title: ready ? t('common:july') : '' },
+    { id: 7, title: ready ? t('common:august') : '' },
+    { id: 8, title: ready ? t('common:september') : '' },
+    { id: 9, title: ready ? t('common:october') : '' },
+    { id: 10, title: ready ? t('common:november') : '' },
+    { id: 11, title: ready ? t('common:december') : '' },
+  ];
+
+  const renderPlantingSeasons = () => {
+    return plantingSeasons.map((season, index) => (
+      <Fragment key={seasons[season - 1].title}>
+        {seasons[season - 1].title}
+        {index === plantingSeasons.length - 2 ? (
+          <> {t('manageProjects:and')} </>
+        ) : index === plantingSeasons.length - 1 ? (
+          '.'
+        ) : (
+          ', '
+        )}
+      </Fragment>
+    ));
+  };
+
+  return (
+    <div className={styles.projectInfoContainer}>
+      <div className={styles.singleInfo}>
+        <SingleProjectInfoItem
+          title={
+            <>
+              {t('manageProjects:abandonment')}
+              <InfoIconPopup
+                height={10}
+                width={10}
+                color={`${'rgba(var(--secondary-divider-color-new))'}`}
+              >
+                <div className={styles.infoIconPopupContainer}>
+                  {t('manageProjects:yearAbandonedInfo')}
+                </div>
+              </InfoIconPopup>
+            </>
+          }
+          itemContent={
+            <>
+              {t('common:approx')} {abandonment}
+            </>
+          }
+        />
+        <SingleProjectInfoItem
+          title={t('projectDetails:firstTreePlanted')}
+          itemContent={
+            <time>
+              {firstTree.length > 0 &&
+                formatDate(
+                  firstTree.split('-')[1].length === 1 ||
+                    firstTree.split('-')[2].length === 1
+                    ? addZeroToDate(firstTree)
+                    : firstTree
+                )}
+            </time>
+          }
+        />
+      </div>
+      <div className={styles.seperator}></div>
+      <div className={styles.singleInfo}>
+        <SingleProjectInfoItem
+          title={t('manageProjects:plantingDensity')}
+          itemContent={
+            <>
+              {plantingDensity}
+              {maxPlantingDensity !== null
+                ? `-${maxPlantingDensity} ${t('manageProjects:treePerHa')}`
+                : ` ${t('manageProjects:treePerHa')}`}
+            </>
+          }
+        />
+        <SingleProjectInfoItem
+          title={t('manageProjects:employees')}
+          itemContent={<>{employees}</>}
```

Comment:
Try to avoid the fragment structure (`<></>`) for a single element. It would be preferable to use a `<p>`, `span`, `div` tag to enclose a single item. Otherwise, you end up with html markup like below, which isn't the easiest to read. 

```
<div class="ProjectInfo_halfInfo___ZcGj">
	<div class="ProjectInfo_infoTitle__9SS4j">Employees</div>
	500
</div>
```

If I were inspecting this code, I may not understand what `500` represents without digging into the code. Enclosing this in `<div class="infoContent">` would make it easier to understand.